src/main/webapp/list/list.css

```diff
@@ -0,0 +1,74 @@
+/* List page */
```

Comment:
nit, can this cc file be lecture-list.css, list is very generic.

Reply:
Done!